assets/messages.yml

```diff
@@ -1044,7 +1044,7 @@ messages:
         - You are in the *correct project* on the bug tracker.
         - You were playing the *latest release version* or the *latest development version* of the game. 
 
-        ~This issue is being temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
+        ~This issue has been closed as {color:#d04437}*Invalid*{color}. Please create a new bug report if you believe this is still a bug~
```

Comment:
Might want to say why the report was closed (a future version was marked as the only affected version)

Reply:
Oops I'm wrong, but the rest of the message needs to be editied to show that adding affected versions will not do anything

Reply:
I've made commit 2da264a but it might need some rewording